pyproject.toml

```diff
@@ -13,7 +13,7 @@ classifiers = [
 description = "Bluesky playground environment for Diamond's htss rigs"
 dependencies = [
     "ophyd",
-    "ophyd-async==0.4.0",
+    "ophyd-async>=0.8.0",
```

Comment:
Should: If you delete the device you should be able to completely remove the dependency on ophyd/ophyd-async